README.md

```diff
@@ -73,4 +72,4 @@ Give a ⭐️ if you like this project!
 - Original design idea by : Cindy Shin  💘
```

Comment:
- Kindly provide links credit to **Cindy Shin** the author of the **original design**, as required by the **Creative Commons License.**